src/utils/images.js

```diff
@@ -0,0 +1,13 @@
+export const formatImagesList = (imagesList) => {
+    const imagesObject = {};
+    const currentImagesList = imagesList.map((obj) => ({ ...obj }));
+
+    if (currentImagesList) {
```

Comment:
The check is meaningless, an empty array will be cast to Boolean as true, see [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean). Let's get rid of her.